app-react/src/jsMain/kotlin/UstadJsDi.kt

```diff
@@ -81,6 +82,10 @@ internal fun ustadJsDi(
         )
     }
 
+    bind<FilterUsernameUseCase>() with singleton {
```

Comment:
ValidateUsernameUseCase must also be bound in DI for web.

Both use cases need bound in app-desktop